aten/src/ATen/native/SummaryOps.cpp

```diff
@@ -52,10 +51,77 @@ Tensor _bincount_cpu_template(
   }
   return output;
 }
+
+// A helper function to get the bin for histogram
+// while each bin is inclusive at the lower end and exclusive at the higher,
+// i.e. [start, end) the last bin is inclusive at both, i.e. [start, end], in
+// order to include maxvalue if exists
+template <typename input_t>
+inline int64_t getbin(input_t x, input_t min, input_t max, int64_t bins) {
+  if (x == max)
+    return bins - 1;
+  return (int64_t)((x - min) * bins / (max - min));
```

Comment:
`static_cast<int64_t>()` instead of C-style cast

Reply:
Done.